swap/services.go

```diff
@@ -47,6 +47,7 @@ type LightningClient interface {
 	AddPaymentNotifier(swapId string, payreq string, invoiceType InvoiceType)
 	RebalancePayment(payreq string, channel string) (preimage string, err error)
 	CanSpend(amountMsat uint64) error
+	Implementation() string
```

Comment:
I might be missing something where in the code does it actually print Implementation() during runtime?

Reply:
https://github.com/ElementsProject/peerswap/blob/23ed52b4997c4e6cd38615699016cc5a7ea25125/swap/service.go#L437-L447

Reply:
So this is reported over the wire to the peer? If so should that be mentioned in the spec too?

Reply:
Mh, good question. I think that the error message is implementation specific, we have the special case here, that the error message is related to the cln option `--large-channel`. I guess that this is something we should loosely communicate or agree on with other implementations (acinq/eclair to be named) instead of enforcing it by spec. The reason for the informative error is to avoid confusion if the swap partner is a different implementation (lnd currently).